plio/utils/covariance.py

```diff
@@ -1,7 +1,7 @@
 import numpy as np
 import math
 
-def compute_covariance(lat, lon, rad, latsigma=10., lonsigma=10., radsigma=15., semimajor_axis=1737400.0):
+def compute_covariance(lat, lon, rad, latsigma=10., lonsigma=10., radsigma=15., semimajor_axis=None):
```

Comment:
Is there a reason why this was defaulting to the moon?

Reply:
I would imagine whoever wrote this originally just happened to be working with the moon. Otherwise, probably threw a dart a picture of the solar system and chose the nearest planet.